content/fr/docs/setup/pick-right-solution.md

```diff
@@ -0,0 +1,302 @@
+
+[Les solutions cloud clés en main](#turnkey-cloud-solutions) ne demandent que peu de commande pour déployer et couvrent un large panel de 
+ fournisseurs de cloud. [Les solutions clés en main pour cloud privé](#on-premises-turnkey-cloud-solutions) possèdent la simplicité des solutions cloud clés en main combinées avec la sécurité de votre propre réseau privé.
+
+Si vous avez déja un moyen de configurer vos resources, utilisez [kubeadm](/docs/setup/independent/create-cluster-kubeadm/) pour facilement
+déployer un cluster grâce à une seule ligne de commande par machine.
+
+[Les solutions personnalisées](#custom-solutions) varient d'instructions pas à pas, à des conseils relativement généraux pour déployer un 
+cluster Kubernetes en partant du début.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Solutions locales
+
+* [Minikube](/docs/setup/minikube/) est une méthode pour créer un cluster Kubernetes local à noeud unique pour le développement et le test. L'installation est entièrement automatisée et ne nécessite pas de compte de fournisseur de cloud.
+
+* [Docker Desktop](https://www.docker.com/products/docker-desktop) est une
+application facile à installer pour votre environnement Mac ou Windows qui vous permet de
+commencer à coder et déployer votre code dans des conteneurs en quelques minutes sur un nœud unique Kubernetes.
+
+* [Minishift](https://docs.okd.io/latest/minishift/) installe la version communautaire de la plate-forme d'entreprise OpenShift 
+de Kubernetes pour le développement local et les tests. Il offre une VM tout-en-un (`minishift start`) pour Windows, macOS et Linux,
+ le `oc cluster up` containerisé (Linux uniquement) et [est livré avec quelques Add Ons faciles à installer] (https://github.com/minishift/minishift-addons/tree/master/add-ons).
+
+* [MicroK8s](https://microk8s.io/) fournit une commande unique d'installation de la dernière version de Kubernetes sur une machine locale
+pour le développement et les tests. L'installation est rapide (~30 sec) et supporte de nombreux plugins dont Istio avec une seule commande.
+
+* [IBM Cloud Private-CE (Community Edition)](https://github.com/IBM/deploy-ibm-cloud-private) peut utiliser VirtualBox sur votre machine
+pour déployer Kubernetes sur une ou plusieurs machines virtuelles afin de développer et réaliser des scénarios de test. Cette solution
+peut créer un cluster multi-nœuds complet.
+
+* [IBM Cloud Private-CE (Community Edition) sur Linux Containers](https://github.com/HSBawa/icp-ce-on-linux-containers) est un script IaC (Infrastructure as Code) basé sur Terraform/Packer/BASH pour créer un cluster LXD à sept nœuds (1 Boot, 1 Master, 1 Management, 1 Proxy et 3 Workers) sur Linux Host.
+
+* [Kubeadm-dind](https://github.com/kubernetes-sigs/kubeadm-dind-cluster) est un cluster Kubernetes multi-nœuds (tandis que minikube est
+un nœud unique) qui ne nécessite qu'un docker-engine. Il utilise la technique du docker-in-docker pour engendrer le cluster Kubernetes.
```

Comment:
You can maybe use simply `créer` instead of `engendrer`.

Reply:
maybe "deployer", thanks !

Reply:
+1 pour `déployer` aussi.